Templates/Flow Coordinator.xctemplate/___VARIABLE_flowCoordinatorIdentifier___FlowCoordinator.swift

```diff
@@ -30,11 +30,8 @@ final class ___VARIABLE_flowCoordinatorIdentifier___FlowCoordinator: NavigationS
 }
 
 extension ___VARIABLE_flowCoordinatorIdentifier___FlowCoordinator {
-    enum Destination: String, Hashable, Identifiable {
+    @EnumIdentable
+    enum Destination: Hashable, Identifiable {
```

Comment:
I still think we should make the `@EnumIdentable` add the conformance. I don't think it's MUST-HAVE at this point, but please, file an Issue to this effect.